_maps/map_files/generic/CentCom_skyrat_z2.dmm

```diff
@@ -22073,10 +22072,6 @@
 /obj/effect/landmark/latejoin,
 /turf/open/floor/iron,
 /area/centcom/interlink)
-"wjm" = (
-/obj/machinery/vending/dorms,
```

Comment:
why are you removing ALL the dorm vendors?

Reply:
pretty sure that the 'dorm vending maching is the kinkmate.